packages/ListBox/src/components/Label/Label.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useListBox from "../../useListBox";
+import labelStyles from "./Label.styles";
+
+const propTypes = {
+  placeholder: PropTypes.string,
+};
+
+let lastKnownLabel = "";
+
+export default function Label(props) {
+  const { placeholder } = props;
+  const [state] = useListBox();
+  const { selectedOptions, isMulti, options } = state;
+
+  if (!isMulti && selectedOptions.length && state.options[selectedOptions[0]].preventDefaultOnSelect) {
+    if (!lastKnownLabel) return placeholder;
+
+    return lastKnownLabel;
+  }
+
+  function getListboxLabelForMulti() {
+    const optionsLength = selectedOptions.length;
+
+    const label = selectedOptions
+      .map(index => {
+        const option = options[index];
+        if (typeof option.content === "string") {
+          return option.content;
+        }
+
+        if (option.label) {
+          return option.label;
+        }
+
+        throw Error(
+          `The trigger label on the ListBox needs that either <ListBox.Option> children are typeof string
+        or a label prop is add to the <ListBox.Option label='my description'> component`
+        );
+      })
+      .join(", ");
+
+    const lastKnownLabel =
+      optionsLength > 1 ? (
+        <span css={labelStyles}>
```

Comment:
I'm noticing here you say:
```
<span css={...}>
  some children
</span>
```
but in other cases you do it like
```
<StyledSpan>
  some children
</StyledSpan>


//StyledSpan.style.js
export const StyledSpan = styled.span`
  ...
`
```

I guess these are equivalent, just different syntaxes? And wondering when to use which?